i18n/vscode-language-pack-uk-UA/CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Change Log
+All notable changes to the "vscode-language-pack-ru" language pack will be documented in this file.
+
+## [Released]
+* May 5, 2021 - Release for VS Code 1.56
```

Comment:
I don't see how this release history has anything to do with this language pack. Please reset it to `* July 12, 2023 - Initial release for VS Code 1.56`